CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 # Change Log
 All notable changes to the "vscode-azurearmtools" extension will be documented in this file.
 
+Version 0.4.1 (05/07/2018)
+
+- Add support for JSONC
```

Comment:
I would add a link to the spec/project here, and an example of extra capability it unlocks. 
Searching the web for JSONC, I see JSON+Comments, JSON with C-type reference counting, and JSON-Compressed. 
I personally expect the first, but a little confirmation would be nice. :)

Reply:
Improved.